example/react/src/App.js

```diff
@@ -1,19 +1,21 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState } from 'react'
 import { GoTrueClient } from '@supabase/gotrue-js'
 import './tailwind.output.css'
 
-const auth = new GoTrueClient()
+const GOTRUE_URL_AUTOCONFIRM = 'http://localhost:9998'
+
+const auth = new GoTrueClient({
+  url: GOTRUE_URL_AUTOCONFIRM,
+})
 
 function App() {
-  let [session, setSession] = useState(null)
+  let [session, setSession] = useState(auth.session())
```

Comment:
If the previous session is in local storage and the token DOESNT need refreshing, this will be a full session

If the previous session is in local storage and the token DOES need refreshing, this will be null (because refreshing the token takes an API call)

Reply:
This could cause some flicker - the user might have a valid session, but it requires an API request to get it (if users haven't set up their apps to wait for a valid session)

Reply:
> If the previous session is in local storage and the token DOES need refreshing, this will be null (because refreshing the token takes an API call)

How do we handle the case where the token needs to be refreshed, but fails because of an invalid refresh token? I've noticed it happening when users are logging in across multiple devices\tabs.